packages/mdc-select/foundation.js

```diff
@@ -155,6 +152,9 @@ export default class MDCSelectFoundation extends MDCFoundation {
     if (this.selectedIndex_ >= 0) {
       selectedTextContent = this.adapter_.getTextForOptionAtIndex(this.selectedIndex_).trim();
       this.adapter_.setAttrForOptionAtIndex(this.selectedIndex_, 'aria-selected', 'true');
+      this.adapter_.floatLabel(true);
+    } else {
+      this.adapter_.floatLabel(false);
```

Comment:
This should only called if the menu is not opened, so that the label is always floating if the menu is opened. 
```
if (this.adapter_.isMenuOpen()) {
    this.adapter_.floatLabel(false);
}
```